packages/redux-subspace-loop/test/integration-spec.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2017, IOOF Holdings Limited.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { createStore } from 'redux'
+import { Cmd, loop, install, combineReducers } from 'redux-loop'
+import { namespaced } from '../src'
+import { subspace } from 'redux-subspace'
+
+describe('integration tests', () => {
```

Comment:
I would like to see `should work with namespaced single subspace` and `should work with namespaced nested subspaces` integration tests as well.

Reply:
Will add them today :)